goentri.com.neo.json

```diff
@@ -0,0 +1,52 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "neo",
+  "serviceName": "Neo",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows the user to easily set up domain using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com",
+  "variableDescription": "domainkeyHost is the Dkim host and keyValue is the dkim value, mxPointsto is to where the email is pointed, domainVaerifcation is the domain verification value",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "TXT",
+      "host": "%domainkeyHost%",
+      "data": "%keyValue%",
+      "groupId": "a1",
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "%mxPointsto1%",
+      "groupId": "a2",
+      "priority": 10,
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "%mxPointsto2%",
+      "groupId": "a2",
+      "priority": 20,
+      "ttl": 3600
+    },
+    {
+      "type": "SPFM",
+      "host": "@",
+      "spfRules": "include:spf0001.neo.space",
+      "groupId": "a3"
+    },
+    {
+      "type": "TXT",
+      "host": "@",
+      "data": "%domainVerifcation%",
```

Comment:
It's fixed, thank your feedback.

Reply:
I mean not in the description but actually here

Reply:
I'm sorry I just mention I would use the draft and I didn't. I'm going to try to remember that option next time.

Reply:
What I mean that there is a typo in this variable
```
  "data": "%domainVerifcation%",
```

Reply:
Yea, I got it. This is solved.

"data": "%domainVerification%"

 I was just apologizing because I didn't use the draft option to create this PR as you required previously.